src/schema/objects/columns.yaml

```diff
@@ -391,6 +462,12 @@ reference__ieeg:
     - type: string
       enum:
         - n/a
+reference_atlas:
```

Comment:
The “reference atlas” if you visually verify the area is basically the atlas you looked at before you nod and said “eh, good enough”. This seems at once an overly detailed (does this really matter? the coordinates are commonly given with sub-mm precision) and underdetermined (what coordinates did you look at in the atlas?). I think it's best just to drop this.